### DISCUSSION TITLE
mock fake data?

### DISCUSSION THREAD

**Discussion by NMTuan** [directus/directus#14505] [Feature Requests]
Sometimes, In development stage, i need lots of fake data. 
If directus can create with one click. I think it's great.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
